models/normalize/staging/normalize__stg_unmapped_immunization.sql

```diff
@@ -0,0 +1,41 @@
+{{ config(
+     enabled = ( ( var('enable_normalize_engine', False) == True  or  var('enable_normalize_engine', False) == "unmapped") and
+                   var('claims_enabled', var('clinical_enabled', var('tuva_marts_enabled', False)))
+               ) | as_bool
+   )
+}}
+
```

Comment:
Remove claims_enabled var since this should only be enabled for clinical.

```suggestion
{{ config(
     enabled = ( ( var('enable_normalize_engine', False) == True  or  var('enable_normalize_engine', False) == "unmapped") and
                   var('clinical_enabled', var('tuva_marts_enabled', False)))
               ) | as_bool
   )
}}

```